.github/workflows/ci.yml

```
@@ -47,7 +64,8 @@ jobs:
           apt-get update
           apt-get install --allow-unauthenticated -y mysql-client
```

@adrianomitre On CI there is such error:
> ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)

Following [this answer](https://stackoverflow.com/a/15039113) I suggest to change this line:
```suggestion
          apt-get install --allow-unauthenticated -y mysql-server
```

If that fix would work then probably we should change `step` name to `Install MySQL Server`.

---

@Eiji7 could you please point out in which commit, job, step such error ocurred?

I could not find it on step `Run MYSQL_URL=root:root@mysql ECTO_ADAPTER=myxql mix test` of the [job `test-mysql (8.0)` of commit 1debbfd9](https://github.com/elixir-ecto/ecto_sql/pull/242/checks?check_run_id=910137830).

---

@Eiji7 Also, it seems that MySQL server is running on a [service container](https://docs.github.com/en/actions/configuring-and-managing-workflows/about-service-containers), running `mysql:${{ matrix.mysql_version }}` image (see https://github.com/elixir-ecto/ecto_sql/pull/242/files#diff-e9f950f17198d3d5e3122a44230a09b9L29).